### DISCUSSION TITLE
Source config files...

### DISCUSSION THREAD

**Discussion by Bersaker** [Jyers/Marlin#1594] [Q&A] [answered]
Hallo. war der nicht in den KonfigurationenDateien in denden in densen von daden ordnnern? Warum IST Jyers Config Seperat?

![Screenshot_20211107-162213_Total Commander](https://user-images.githubusercontent.com/62776321/140651452-c6ec630a-d733-4f0a-a1d3-5817972808b0.jpg)

**Comment by Cal-LF** [comment, marked as answer]
Hello. Wasn't that in the configuration files in the files in those of the folders? Why IS Jyers Config Separate?

I think it is available for those who would like to view his personal config files.

**Reply by Jyers** [reply]
Yes, the Jyers's Config folder just has my personal config in it for those interested. Most people should start from the templates and disregard my config.

**Reply by Bersaker** [reply]
Ok. Thank you @Jyers ...
I think so far the basic config should be the same, except for the offsets etc.

**Reply by Cal-LF** [reply]
@Bersaker I believe @Jyers has linear advance active in his configs.

**Reply by Bersaker** [reply]
@Cal-LF du you know does the ender 3v2 actually support linear advance with the 4.2.2 board ?

**Reply by Cal-LF** [reply]
@Bersaker The 422 and 427 boards does not support linear advance without modifying. See #707
